bindings/java/src/test/java/org/openucx/jucx/UcpListenerTest.java

```diff
@@ -127,7 +127,7 @@ public void testConnectionHandler() {
         UcpRequest sent = serverToClient.sendStreamNonBlocking(
             ByteBuffer.allocateDirect(UcpMemoryTest.MEM_SIZE), null);
         UcpRequest recv = clientToServer.recvStreamNonBlocking(
-            ByteBuffer.allocateDirect(UcpMemoryTest.MEM_SIZE), null);
+            ByteBuffer.allocateDirect(UcpMemoryTest.MEM_SIZE), 0, null);
```

Comment:
how does it work? looks like it will call recv non blocking with length=0 because remaining is 0?

Reply:
No, `0` is a flag. There's 2 method - big one with all the fields, and the short one with buffer and callback only. In this case, size would be determined from buffer capacity.

Reply:
right, missed it's a flag